src/composables/useRuleForm.ts

```diff
@@ -171,6 +172,14 @@ export function useRuleForm(policies: Ref<Policy[]>, rule?: Ref<Rule | undefined
     if (!validationCheck.valid) {
       validationErrors.value.set('policy', String(validationCheck.errMessage))
     }
+    validationCheck = validateUciName(name.value, 12)
+    if (!validationCheck.valid) {
+      validationErrors.value.set('name', String(validationCheck.errMessage)),
+        validationErrors.value.set(
+          'num',
+          String((validationCheck.i18Params as { num?: number })?.num)
+        )
+    }
```

Comment:
To see how to handle the translation process of these cases in a more clean way: https://github.com/NethServer/nethsecurity-ui/blob/898bc8593bc8cbdb212ff84e604d0adf4af9ad77/src/components/standalone/interfaces_and_devices/ConfigureDeviceDrawer.vue#L663